docs/html-css/week-1/lesson.md

```diff
@@ -174,19 +174,19 @@ When creating a web page, we should always think about how to structure the cont
 
 ### Sectioning Content
 
-We can group elements with 'sectioning' tags.
+We can apply structure to a web page using 'sectioning' elements.
 
 <img src={require('!file-loader!../assets/webpage-structure.png').default}
 alt="Wireframe of a web page with <header> at the top, <main> at the middle, <footer> at the bottom, and <aside> at the right"/>
 
 The image above shows a common layout of a web page.
 We can use specific HTML elements for each of these sections.
 
-- We can use a `<header>` tag for the header content
-- We can use a `<footer>` tag for the footer content
-- We can use a `<main>` tag for the main content of the page
-- Sidebar content can go in an `<aside>` tag
-- If there is a list of links, such as in the `<header>`, they can go in a `<nav>` tag
+- We can use a `<header>` element for the header content
```

Comment:
I've got no objection to using element instead of tag. It is the spec after all. We might want to mention tag somewhere to increase their Googlefu.

Reply:
There are mentions of tags elsewhere on the page that are arguably reasonable uses (thinking of the tabular content section). Does that fulfil this concern?

Reply:
Yes, I think so 👍